src/internal/polyfills/Promise.any.js

```diff
@@ -0,0 +1,36 @@
+import { map } from 'ramda';
+
+import resolveP from '../../resolveP';
+
+export class AggregatedError extends Error {
+  constructor(exceptions) {
+    super();
+
+    this.exceptions = exceptions;
+  }
+}
+
+const anyPolyfill = iterable => {
+  const exceptions = [];
+
+  return new Promise((resolve, reject) => {
+    const onReject = e => {
+      exceptions.push(e);
+
+      if (exceptions.length === iterable.length) {
+        reject(new AggregatedError(exceptions));
+      }
+    };
+
+    map(
```

Comment:
Suggestion without epxlicit `{}`. Formatting will be handled by prettier.
```js
map(p => resolveP(p).then(resolve).catch(onReject), [...iterable])

Reply:
I removed the braces, but prettier saved the formating